src/components/WarningAnimateMessage/index.js

```diff
@@ -129,4 +127,9 @@ const styles = StyleSheet.create({
   }
 });
 
+WarningAnimatedMessage.propTypes = {
+  isShow: PropTypes.bool,
+  top: PropTypes.number,
+  left: PropTypes.number
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

PropTypes have been added for `isShow`, `top`, and `left` props. This is a good practice as it helps ensure type safety for the props and can prevent potential bugs due to incorrect prop types.
```diff
+ WarningAnimatedMessage.propTypes = {
+   isShow: PropTypes.bool,
+   top: PropTypes.number,
+   left: PropTypes.number
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->